thrusterCalibration/runThrusters.ino

```diff
@@ -26,15 +29,21 @@ enum Commands {
   BACKWARDSPEEDUP   , // 7
   BACKWARDSPEEDDOWN   // 8
 };
-
+//intialize ??
 int currentGroup1   = 0;
-int currentCommand1 = 0;
+int currentCommand1 = PAUSE;
 int currentGroup2   = 0;
-int currentCommand2 = 0;
+int currentCommand2 = PAUSE;
 int currentGroup3   = 0;
-int currentCommand3 = 0;
-
-
+int currentCommand3 = PAUSE;
+
+/*
+Thurster Value Range
+1500 +/- 30 is stop
+1900 - 1500 +/- 30 is forward
+1500 - 1100 +/- is backward
+*/
+//Is this values there because the singal was noisy
```

Comment:
'Thurster' -> Thruster.

Change to:
1500: Stop
1501-1900: Forward
1100-1499: Backward

No it was not due to noisy signal.